coalib/parsing/CliParsing.py

```diff
@@ -65,6 +65,12 @@ def parse_cli(arg_list=None,
                                   origin,
                                   line_parser)
         else:
+            # if 'Bear' suffix is not already present then add it here
+            if arg_key == 'bears' and isinstance(arg_value, list):
+                arg_value = [
+                    Bear + 'Bear' if Bear and not Bear.endswith('Bear')
+                    else Bear for Bear in arg_value]
```

Comment:
Also: Why
`if bear and not bear.endswith(...)`, so why the `if bear` part? Can `bear` be `None` or an empty string?